src/gui/helpers/mod.rs

```diff
@@ -2,12 +2,14 @@ pub mod debug_helper;
 pub mod inventory_helper;
 pub mod main_helper;
 pub mod nav_helper;
+pub mod shop_helper;
```

Comment:
As a side note (not 100% certain how rust visibility works), but isn't `pub mod` AND `pub use self::...` redundant? Don't think it makes any difference. Could just have regular `mod`, and then `pub use self::{debug_helper::DebugHelper, ...` (don't think anything else is needed/used outside?)

Reply:
Honestly in this case there's no reason to have pub on either of them since they aren't going to be used outside of the `Gui` module.

Reply:
but the self primarily exists to make:
```rust
impl Default for GuiHelpers {
    fn default() -> Self {
        let mut helpers: HashMap<String, Box<dyn GuiHelper>> = HashMap::new();
        helpers.insert(nav_helper::NAME.to_owned(), NavHelper::create());
        helpers.insert(main_helper::NAME.to_owned(), MainHelper::create());
        helpers.insert(title_helper::NAME.to_owned(), TitleHelper::create());
        helpers.insert(debug_helper::NAME.to_owned(), DebugHelper::create());
        helpers.insert(shop_helper::NAME.to_owned(), ShopHelper::create());
        helpers.insert(inventory_helper::NAME.to_owned(), InventoryHelper::create());
        Self { helpers }
    }
}
```
this fit on one line and be readable without doubling the code size